backend/clubs/models.py

```diff
@@ -1194,6 +1194,27 @@ def __str__(self):
         return self.user.username
 
 
+class ClubApplication(models.Model):
+    """
+    Represents an application to a club.
+    """
+
+    club = models.ForeignKey(Club, on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, blank=True)
```

Comment:
I would recommend using a `TextField` instead of a `CharField` for unlimited length! The performance impact is negligible and who knows how long people would want to make their titles.